examples/options/src/track_finding.cpp

```diff
@@ -102,6 +107,8 @@ std::unique_ptr<configuration_printable> track_finding::as_printable() const {
         std::to_string(m_config.max_num_skipping_per_cand)));
     cat->add_child(std::make_unique<configuration_kv_pair>(
         "PDG number", std::to_string(m_pdg_number)));
+    cat->add_child(std::make_unique<configuration_kv_pair>(
+        "Debug output", std::format("{}", m_config.verbose)));
```

Comment:
If you call it "debug" here as well, then let's make the variable name `debug`...